make fails with no params

``` /bin/bash
$ git log
commit 569f1bf2ac2d1f85311d27a8ba4b84aeb44be8c1
Author: Jordan Sissel <jls@semicomplete.com>
Date:   Fri Jan 11 22:56:38 2013 -0800

    - use cabin 0.6.0's new logger.pipe method.

$ make
rm -rf .yardoc
sh notify-failure.sh ruby test/all.rb
/usr/lib/ruby/vendor_ruby/1.8/rubygems/custom_require.rb:36:in `gem_original_require': no such file to load -- minitest/spec (LoadError)
    from /usr/lib/ruby/vendor_ruby/1.8/rubygems/custom_require.rb:36:in `require'
    from test/all.rb:2
make: *** [test] Error 1
```

--- a/fpm.gemspec
+++ b/fpm.gemspec
@@ -46,6 +46,7 @@ Gem::Specification.new do |spec|
 
   spec.add_development_dependency("rspec") # license: MIT (according to wikipedia)
   spec.add_development_dependency("insist", "~> 0.0.5") # license: ???
+  spec.add_development_dependency("minitest")
 
   spec.files = files
   spec.require_paths << "lib"

--- a/lib/fpm/package.rb
+++ b/lib/fpm/package.rb
@@ -487,7 +487,7 @@ class FPM::Package
     if !File.directory?(File.dirname(output_path))
       raise ParentDirectoryMissing.new(output_path)
     end
-    if File.exists?(output_path)
+    if File.file?(output_path)
       if attributes[:force?]
         @logger.warn("--force flag given, overwriting package at #{output_path}")
       else

